eng/Directory.Build.Data.targets

```diff
@@ -10,15 +10,19 @@
     <Error Condition="'@(MissingTargetFrameworks)' != ''" Text="Missing required target frameworks '@(MissingTargetFrameworks)'. Please ensure you add those frameworks." />
   </Target>
 
-  <!-- Set PackageProjectUrl to the package README for Client Libraries -->
-  <Target Name="SetPackageProjectUrl" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsClientLibrary)' == 'true' and '$(IsTestProject)' != 'true' and
+  <!-- Set PackageProjectUrl and PackageReleaseNotes to the package README.md and CHANGELOG.md respectively for DataPlane Libraries -->
+  <Target Name="SetPackageProjectUrlandReleaseNotes" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsDataPlaneProject)' == 'true' and '$(IsTestProject)' != 'true' and
    '$(IsSamplesProject)' != 'true' and '$(IsTestSupportProject)' != 'true' and '$(CommitSHA)' != ''">
     <PropertyGroup>
       <CurrentCsprojPath>$(MSBuildProjectDirectory)</CurrentCsprojPath>
       <DirectoryPartofPath>$(CurrentCsprojPath.Substring($(CurrentCsprojPath.LastIndexOf("\sdk\"))))</DirectoryPartofPath>
       <DirectoryPartofPath>$(DirectoryPartofPath.Remove($(DirectoryPartofPath.LastIndexOf("\src"))))</DirectoryPartofPath>
       <PackageProjectUrl>$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/README.md</PackageProjectUrl>
+      <PackageReleaseNotes Condition="'$(PackageReleaseNotes)' == '' and 'Exists('$(RepoRoot)/$(DirectoryPartofPath)/CHANGELOG.md')''">$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/CHANGELOG.md<PackageReleaseNotes>
     </PropertyGroup>
+    <ItemGroup Condition="Exists('$(RepoRoot)/$(DirectoryPartofPath)/CHANGELOG.md')">
```

Comment:
I don't remember if we had an issue for this but didn't we also want to include the README.md in the package as well?

Reply:
I was not aware of that. I can add it here if needed.

Reply:
Yes I think it would be nice to include the README in the package as well in the same fashion.

Reply:
Included